src/log4net/Appender/FileAppender.cs

```diff
@@ -779,16 +782,26 @@ public override void ActivateOptions()
       {
         if (m_mutex == null)
         {
-          string mutexFriendlyFilename = CurrentAppender.File
-              .Replace("\\", "_")
-              .Replace(":", "_")
-              .Replace("/", "_");
+          if (CurrentAppender is not null)
```

Comment:
Examined but prefer to keep this one, the outer if has no else{Error()} because a null CurrentAppender can be ignored, whereas the inner if has an error output.